src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrent.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```

Comment:
Hi. I noticed this. Likely do with my editor/git settings? I've just gone back and rebased. It appears to be fixed now.